.pylintrc

```diff
@@ -70,7 +70,14 @@ disable=missing-docstring,
       wrong-import-order, # Leave this up to isort
       bad-continuation, # Leave this up to black
       line-too-long, # Leave this up to black
```

Comment:
Can these all be removed after upgrade? Looking at the black docs, the config for compatibility should be pretty minimal https://black.readthedocs.io/en/stable/compatible_configs.html#pylint

Reply:
I haven't looked into it but I'm for removing rules if we can automate the fixes with black+isort. I really really miss gofmt :(